docs/.gitignore

```diff
@@ -0,0 +1,5 @@
+Gemfile.lock
```

Comment:
Same as above, it'd be nice to just exclude a single dir instead of tracking each file, which may change as tools evolve.

Reply:
I don't think that's doable with how Jekyll works. It wants to serve up things in this directory. Perhaps it has support for out of tree builds that I'm unaware of though.

Reply:
I'm pretty sure this won't work. I'd have to figure out how to change how GitHub pages invokes Jekyll, which I doubt is possible. I could make it build to a different directory locally, but we would still need these .gitignores for the GitHub action that automatically commits when the docs change.

Reply:
Moved all docs related .gitignores to `docs/.gitignore` in 0a924a7d.

Reply:
Nope, this breaks stuff in the GitHub action, and it'd be complicated to fix it. I'm going to move all the ignores into the main directory instead.